build.gradle

```diff
@@ -432,8 +436,16 @@ allprojects {
     options.addStringOption('Xwerror', '-html5')
     options.encoding = 'UTF-8'
   }
+
+  plugins.withType(JavaPlugin) {
+    tasks.withType(JavaCompile) {
+      it.dependsOn(rootProject.tasks.named('spotlessCheckFast'))
+    }
+  }
 }
 
+check.dependsOn spotlessCheckFast
```

Comment:
can you move it line number 470 i.e. `check.dependsOn('checkPluginAPIChanges', 'checkMavenCoordinateCollisions', 'spotlessCheckFast')` ?